engine/release-notes.md

```diff
@@ -82,6 +82,52 @@ compatibility reasons.
 
 * Fix jsonfile logger: follow logs stuck when `max-size` is set and `max-file=1`. [docker/engine#378](https://github.com/docker/engine/pull/378)
 
+### Known Issues
+
+#### New
+
+* `DOCKER-USER` iptables chain is missing for [docker/for-linux#810](https://github.com/docker/for-linux/issues/810).
+  Users cannot perform additional container network traffic filtering on top of
+  this iptables chain. You are not affected by this issue if you are not
+  customizing iptable chains on top of `DOCKER-USER`.
+     - **Workaround:** Insert the iptables chain after the docker daemon starts.
+       Below is an example.
+       ```
+       iptables -N DOCKER-USER
+       iptables -I FORWARD -j DOCKER-USER
+       iptables -A DOCKER-USER -j RETURN
+       ```
+
+#### Existing
+
+* In some circumstances, in large clusters, docker information might, as part of the Swarm section,
+  include the error `code = ResourceExhausted desc = grpc: received message larger than
+  max (5351376 vs. 4194304)`. This does not indicate any failure or misconfiguration by the user,
+  and requires no response.
+* Orchestrator port conflict can occur when redeploying all services as new. Due to many swarm manager
+  requests in a short amount of time, some services are not able to receive traffic and are causing a `404`
+  error after being deployed.
+     - **Workaround:** restart all tasks via `docker service update --force`.
+
+* Traffic cannot egress the HOST because of missing Iptables rules in the FORWARD chain
+  The missing rules are :
+     ```
+     sbin/iptables --wait -C FORWARD -o docker_gwbridge -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
```

Comment:
missing leading `/`

Reply:
why do we need these, these have been fixed https://github.com/docker/orca/pull/17254